build.xml

```diff
@@ -28,7 +31,9 @@
         <var file="whisk.properties" />
         <ant target="scanCode" />
         <exec executable="./gradlew" failonerror="true">
+            <env key="DOCKER_HOST" value="${main.docker.endpoint}" />
```

Comment:
Having tried this and discussed with @markusthoemmes  - will use --host rather than DOCKER_HOST so that it it easier to retry commands from the shell should things go wrong and also to make it clearer in the logs which host one is running docker commands against.